src/lib/tabs/tab.ts

```diff
@@ -66,6 +66,12 @@ export class MatTab extends _MatTabMixinBase implements OnInit, CanDisable, OnCh
    */
   @Input('aria-labelledby') ariaLabelledby: string;
 
+  /**
+   * Unique identifier for each tab.
+   * Used to differentiate which tab is being changed or switched to.
+   */
+  @Input('unique-name') uniqueName: string;
```

Comment:
I was thinking of using `name`, but, I thought it would be misinterpreted as `name` in html attribute.
Is it fine?

I will make changes to the rest of the comments.